Formula/ngt.rb

```diff
@@ -7,7 +7,7 @@ class Ngt < Formula
 
   livecheck do
     url :stable
-    strategy :github_latest
+    regex(/^v?(\d+(?:\.\d+)+)$/i)
```

Comment:
Might this also not pick up tagged pre-releases?

Reply:
Only if the pre-releases are also formatted like `1.2`/`v1.2` instead of `1.2-beta`, `1.2-rc`, etc. Any versions like `1.2-beta`, `1.2-rc`, etc. are omitted by the standard regex for Git tags like `1.2`/`v1.2`, as the end of the regex is restricted using `$` (so nothing can come after the version).

There are certainly some repositories that will tag a version like `1.2.3` and then mark it as "pre-release" on GitHub but this repository doesn't appear to be one of those. Usually those repositories are also better about ensuring the "latest" release works properly, unlike here.